hs628_tools/Visualization.py

```diff
@@ -0,0 +1,294 @@
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+import pandas as pd
+import sys 
+import plotly.plotly as py
+import plotly.graph_objs as go
+#plotly.tools.set_credentials_file(username='hhowell', api_key='BRP1aHhfTHwkqnvalVQL')
+from sklearn import datasets, linear_model
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import mean_squared_error,mean_absolute_error, r2_score
+###This is importing a class from Issue #11 Test module. We used the ratio function to determine the test size ratio for the train_test_split function. 
+#from Test import data_split as ds
+
+
+###Bar Plot-Count
+#This function counts all instances in a variable and creates a barplot to display the results 
+#@param: The variable to be counted, the x_axis label, title of the graph
+#@return: A count bar plot of the variable inputted 
+def count_and_print(variable, x_label, title):
```

Comment:
```variable``` is a rather nondescript name for a variable.  And although we could reasonably infer that ```x_label``` and ```title``` should be strings, it should be clearly identified (by the name or in the docstring) what type this other parameter should be.